src/main/kotlin/com/chattriggers/ctjs/utils/SSLUtil.kt

```diff
@@ -0,0 +1,27 @@
+package com.chattriggers.ctjs.utils
+
+import java.security.KeyStore
+import javax.net.ssl.KeyManagerFactory
+import javax.net.ssl.SSLContext
+import javax.net.ssl.TrustManagerFactory
+object SSLUtil {
```

Comment:
This doesn't need it's own class imo. It should just be a lazy field on the CTJS class:

```kotlin
class CTJS {
    companion object {
        internal val sslContext by lazy {
            // ...
        }
    }
}
```